src/Rules/BannedNodesErrorBuilder.php

```diff
@@ -30,7 +31,7 @@ public function __construct(private readonly bool $nonIgnorable)
     public function buildError(
         string $errorMessage,
         string $errorSuffix
-    ): RuleError {
+    ): IdentifierRuleError {
```

Comment:
Still have `RuleError` in https://github.com/phpstan/phpstan-src/blob/2.0.1/src/Rules/RuleErrorBuilder.php#L260

Reply:
I've a PHPStan warning:

> Method Ekino\PHPStanBannedCode\Rules\BannedNodesRule::processNode() should return list<PHPStan\Rules\IdentifierRuleError> but returns array{PHPStan\Rules\RuleError}

Because of the `identifier` method call, the return type change https://github.com/phpstan/phpstan-src/blob/7eef99af807040bc3f2d91fa1bdf28329b0c65b2/src/Rules/RuleErrorBuilder.php#L219C18-L221